src/risedevtool/src/task/connector_service.rs

```diff
@@ -33,7 +33,7 @@ impl ConnectorNodeService {
     fn connector_path(&self) -> Result<PathBuf> {
         let prefix_bin = env::var("PREFIX_BIN")?;
         Ok(Path::new(&prefix_bin)
-            .join("connector-node")
+            .join("../../../../ci/connector-node")
```

Comment:
Why you need to modify this line? It needs to be the `$PREFIX_BIN/connector-node` directory.